Dockerfile.h2

```diff
@@ -1,13 +1,16 @@
 FROM maven:3-jdk-8-alpine as builder
+RUN apk update && apk add protobuf
+RUN apk add grpc-java --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```

Comment:
Is this safe to use?

Reply:
i only found precompiled musl compatible grpc-java in the edge/testing channel (https://pkgs.alpinelinux.org/packages?name=grpc-java&branch=edge)

A viable alternative would be to compile on a glibc based system like `maven:3-jdk-8-slim` which is based on debian. This would add some overhead to the compile time as the debian image is significantly larger, but it would save us from the musl hell with grpc.

Reply:
In that case we would not need the changes done to fix musl, downloadng the prebuilt grpc and protobuf bins through maven would be fine in that case.

Reply:
If it isn't available outside of edge channel then it's fine